provision-base.sh

```diff
@@ -60,7 +60,8 @@ install_ssmtp()
 	sed -e "/^root=/ s/postmaster/$TOASTER_ADMIN_EMAIL/" \
 		-e "/^mailhub=/ s/=mail/=haraka/" \
 		-e "/^rewriteDomain=/ s/=\$/=$TOASTER_MAIL_DOMAIN/" \
-		"$BASE_MNT/usr/local/etc/ssmtp/ssmtp.conf.sample" \
+                -i '/^#FromLineOverride=YES/ s/#//' \
+                "$BASE_MNT/usr/local/etc/ssmtp/ssmtp.conf.sample" \
 		> "$BASE_MNT/usr/local/etc/ssmtp/ssmtp.conf" || exit
```

Comment:
At least have this setting in production and does what it should do.
If I don't set it, mail is send as "User &" if I set this to yes: "wordpress@<domainname>"

Fixed the expandtab (it was twice in my .vimrc